llvm/lib/Target/AMDGPU/AMDGPUCallLowering.cpp

```diff
@@ -73,7 +79,7 @@ struct AMDGPUOutgoingValueHandler : public CallLowering::OutgoingValueHandler {
     if (TRI->isSGPRReg(MRI, PhysReg)) {
       LLT Ty = MRI.getType(ExtReg);
       LLT S32 = LLT::scalar(32);
-      if (Ty != S32) {
+      if (Ty != S32 && Ty != LLT::scalar(64)) {
```

Comment:
Shouldn't need to special case the types here

Reply:
Done.